CalibPPS/TimingCalibration/plugins/PPSTimingCalibrationPCLHarvester.cc

```diff
@@ -96,7 +96,7 @@ void PPSTimingCalibrationPCLHarvester::dqmEndJob(DQMStore::IBooker& iBooker, DQM
         (int)detid.arm(), (int)detid.station(), (int)detid.plane(), (int)detid.channel()};
 
     calib_params[key] = {0, 0, 0, 0};
-    calib_time[key] = std::make_pair(0.1, 0.);
+    calib_time[key] = std::make_pair(0., 0.1);
```

Comment:
can these numbers be made not hard-coded? configurable would be the best, but if it doesn't change just make it a const expression